test_ped.py

```diff
@@ -8,7 +8,16 @@
 
 import ped
 from ped.guess_module import guess_module
-
+import importlib
+import os
+
+def test_dir_opening():
+    ped_obj = importlib.import_module('ped')
+    curr_os_environ_ped_od = os.environ["PED_OPEN_DIRECTORIES"]
```

Comment:
There's a built-in pytest fixture for temporarily setting envvars: `env`.

Reply:
Thanks. I'll look into it :)